MockerTests/MockerTests.swift

```diff
@@ -228,4 +228,18 @@ final class MockerTests: XCTestCase {
         Mocker.ignore(ignoredURL)
         XCTAssert(MockingURLProtocol.canInit(with: URLRequest(url: ignoredURL)) == false)
     }
+    
+    func testComposedURLMatch() {
+        let composedURL = URL(fileURLWithPath: "resource", relativeTo: URL(string: "https://host.com/api/"))
+        let simpleURL = URL(string: "https://host.com/api/resource")
```

Comment:
You can force unwrap this one which enables you to remove the XCTFail on line 241